lint/action.yaml

```diff
@@ -61,14 +49,20 @@ runs:
       run: npm ci
       env:
         NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
-    - name: Eslint
-      uses: reviewdog/action-eslint@v1
-      with:
-        github_token: ${{ inputs.github-token }}
-        level: ${{ inputs.level }}
-        fail_on_error: ${{ inputs.fail-on-error }}
-        eslint_flags: ${{ inputs.eslint-flags }}
-        reporter: github-pr-review
+    - name: Run eslint (yarn)
+      if: steps.check_yarn_lock.outputs.files_exists == 'true'
+      shell: bash
+      run: yarn eslint ${{ inputs.eslint-flags }}
+      env:
+        NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
+        NPM_TOKEN: ${{ inputs.npm-token }}
+    - name: Run eslint (npm)
+      if: steps.check_yarn_lock.outputs.files_exists == 'false'
+      shell: bash
+      run: npm run eslint ${{ inputs.eslint-flags }}
```

Comment:
Should this be `npx eslint XXX`? In case users don't have a `eslint` script in the package.json, using `npx` will make it work

Reply:
In that case we also wouldn't need to have one option for npm and one for yarn right?

Reply:
That's true!

Reply:
But then I guess it would always be using the latest version of eslint in CI too no? 💭